Rubberduck.VBEEditor/Variants/VariantComparer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Globalization;
+
+namespace Rubberduck.VBEditor.Variants
+{
+    public static class VariantComparer
+    {
+        public static VariantComparisonResults Compare(object x, object y)
+        {
+            return Compare(x, y, VariantComparisonFlags.NORM_IGNORECASE);
```

Comment:
Should we account for (and honor) the module's `Option Compare` statement here? Obviously `Option Compare Database` would need host-specific logic, but the default/implicit string comparison setting being `Compare Binary`, the default (i.e. ignoring any explicit `Option` statements) would be case-*sensitive*. If `Debug.Print "y" = "Y"` outputs `False` in a test module, then `PermissiveAssert` may be behaving unexpectedly here.

OTOH we can just document it as making consistently (i.e. regardless of any `Option Compare` statement) case-insensitive string comparisons, and carry on 😏

Reply:
As discussed, this change would require more work due to the fact that we must communicate the `Option Compare` to the `IAssert` somehow and that necessitates some kind of global shared state. I think I'd rather address this (if ever) as a separate PR.